src/Stratis.StratisD/Program.cs

```diff
@@ -39,6 +40,7 @@ public static void Main(string[] args)
                     .UseBlockStore()
                     .UseMempool()
                     .UseWallet()
+                    .UseTestFeature()
```

Comment:
You can create a new console app showing off this new feature. It's a bit early to add it to StratisD. :stuck_out_tongue_winking_eye:

Reply:
Yes perhaps call it StratisUiD